lib/xlsxtream/io/zip_tricks_io.rb

```diff
@@ -0,0 +1,26 @@
+require "zip_tricks"
+
+module Xlsxtream
+  module IO
+    class ZipTricksIO
+      def initialize(body)
+        @streamer = ZipTricks::Streamer.new(body)
```

Comment:
This needs to be changed to `::ZipTricks::Streamer` to avoid ambiguous constant lookup after removing the "IO" suffix from the class.